### PR TITLE
Dutch translation improvements

### PR DESCRIPTION
**Pull request by mveunen**
Just some Dutch translation improvements

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7100"></a>

### CODE REVIEW

**Review by basil — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
